concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -83,6 +83,7 @@ resources:
     source:
       uri: https://github.com/alphagov/paas-graphite-statsd-boshrelease.git
       tag_filter: {{cf_graphite_version}}
+      branch: 118272895_bosh20
 
```

Comment:
If the tag points to a commit in the default branch then it can be omitted, so once alphagov/paas-graphite-statsd-boshrelease#5 is merged I think it could be omitted.